core-concepts/wallets/veworld/user-guide/wallet.md

```diff
@@ -0,0 +1,244 @@
+
+Steps for importing a Ledger wallet:
+
+1. In the second section click the **Import wallet** button
+2. In the second section _Hardware Wallet_ click the **Import Wallet** button
+3. Follow the instructions and click the **Ledger** button
+4. Follow the instructions and click the **Continue** button
+5. Select your desired account then click the **Add** button
+{% endtab %}
+{% endtabs %}
+
+### Import an Observable wallet
+
+{% tabs %}
+{% tab title="Mobile" %}
+Steps for importing an observable wallet:
+
+1. From your dashboard tap the top-right wallet icon ![](../../../../.gitbook/assets/veworld/account\_balance\_wallet.svg)
+2. Tap the **Add Wallet** button
+3. Tap the **Observe Wallet** button
+4. Tap **Local wallet**
+5. Paste the desired address, then tap **IMPORT** button
+{% endtab %}
+
+{% tab title="Extension" %}
+Not available yet on the Extension.
+{% endtab %}
+{% endtabs %}
+
+## Export a wallet
+
+{% tabs %}
+{% tab title="Mobile" %}
+Steps for exporting a local wallet:
+
+1. Tap the bottom right icon ![](../../../../.gitbook/assets/veworld/settings.svg) to the **Settings** page
+2. Tap **Privacy and Security** from the menu
+3. Tap **Backup your mnemonic phrase**
+4. Select the local wallet you want to export
+5. Pass the authentication (fingerprint, faceid or password)
+6. Paste or type your saved mnemonic and then tap **VERIFY** button
+7. Read or copy to clipboard your mnemonic
+{% endtab %}
+
+{% tab title="Extension" %}
+Steps for exporting a local wallet:
+
+1. Click the bottom right icon ![](../../../../.gitbook/assets/veworld/settings.svg) to **Settings** page
+2. Click **Privacy and Security** from menu
+3. Click **Backup mnemonic**
+4. Select the local wallet you want to export, then click the **Continue** button
+5. Type your password
+6. Read or copy to clipbaord your mnemonic
+{% endtab %}
+{% endtabs %}
+
+## Manage your wallets
+
+With VeWorld you can handle multiple wallets and multiple accounts.
+
+Feel the benefits of self-organize your wallets as you prefer.
+
+{% tabs %}
+{% tab title="Mobile" %}
+Change the order of your wallets
+
+1. From your dashboard tap the top-right wallet icon ![](../../../../.gitbook/assets/veworld/account\_balance\_wallet.svg)
+2. Tap the icon order-change ![](../../../../.gitbook/assets/veworld/low\_priority.svg)
+3. Move your wallets to your favorite order
+4. Tap the **Save** button
+
+Rename a wallet
+
```

Comment:
I would add an informational note explaining that the operation is not final and that is not unrecoverable. 
As well a as a warning, that the user shall be aware that for recovering the wallet, the mnemonic/private key is required.